app/src/main/AndroidManifest.xml

```diff
@@ -32,7 +32,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/StartupTheme"
-        tools:targetApi="n">
```

Comment:
That's just telling the IDE to not complain if some of the used attributes don't work for all SDKs we support. Hence we should only change that if required to suppress incompatibility warnings, it's a bad idea to do so now.